debezium-core/src/main/java/io/debezium/transforms/UnwrapFromEnvelope.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.transforms.ExtractField;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+
+/**
+ * Debezium generates CDC (<code>Envelope</code>) records that are struct of values containing values
+ * <code>before</code> and <code>after change</code>. Sink connectors usually are not able to work
+ * with a complex structure so a user use this SMT to extract <code>after</code> value and send it down
+ * unwrapped in <code>Envelope</code>.
+ * The functionality is similar to <code>ExtractField</code> SMT but has a special semantics for handling
+ * delete events.
```

Comment:
It is in the docs - with a complex structure so a user use this SMT to extract <code>after</code>